eng/pipelines/libraries/build-job.yml

```diff
@@ -80,7 +81,7 @@ jobs:
 
         - ${{ if eq(parameters.osGroup, 'OSX') }}:
           - script: |
-              brew install pkgconfig icu4c openssl
+              brew install pkgconfig icu4c openssl autoconf automake libtool pkg-config python3
```

Comment:
why do we need these? This is building libraries, not mono.

Reply:
cc @akoeplinger

Reply:
We probably don't. I was just adding dependencies where it seemed fit and didn't notice this is only used for libraries build.